src/lib.rs

```diff
@@ -500,8 +471,8 @@ impl<T: Translation> Mapping<T> {
     /// checks to avoid violating break-before-make requirements.
     ///
     /// It is called automatically by [`activate`](Self::activate).
-    pub fn mark_active(&mut self, previous_ttbr: usize) {
-        self.previous_ttbr = Some(previous_ttbr);
+    pub fn mark_active(&mut self) {
```

Comment:
Can this (and `mark_inactive`, `activate` and `deactivate`) take `&self` rather than `&mut self`, as the `AtomicUsize` provides interior mutability?

Reply:
yes that seems to work